pom.xml

```diff
@@ -30,28 +30,28 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.642.1</jenkins.version>
```

Comment:
Don't worry about copyartifact.
The target version should be upgraded if it's required for features.
`Symbol` support of workflow-cps-plugin is since 2.10, and workflow 2.x targets Jenkins 1.642.3.
Anyway, copyartifact also should support `Symbol` and target 1.642.